src/components/favorites-tab.less

```diff
@@ -5,3 +5,36 @@
   justify-content: flex-end;
   margin-bottom: 20px;
 }
+
+.FavoritesTab-actions {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 5px;
+}
+
+.FavoritesTab-action {
+  .m-box-shadow;
```

Comment:
What would you think about removing the shadow in such cases where the background is white and only adding the `m.-box-shadow` on hover?
This default shadow makes sense when the background is not white I feel, but it seems a bet strange on white.
If you feel the same, then maybe we can change it on the `documents` page too 🤔

Reply:
Sure, good idea, we could extract them 👍

Reply:
I'll make a ticket then :+1: